user-interface/src/main/java/life/qbic/datamanager/views/projects/project/ProjectMainLayout.java

```diff
@@ -81,7 +84,22 @@ public void beforeEnter(BeforeEnterEvent beforeEnterEvent) {
   private void setProjectNameAsTitle(ProjectId projectId) {
     projectInformationService.find(projectId)
         .ifPresent(
-            project -> projectTitle.setText(project.getProjectIntent().projectTitle().title()));
+            project -> {
+              projectTitle.removeAll();
+
+              Text projectCode = new Text(project.getProjectCode().value() + " - ");
+              Text projectName = new Text(project.getProjectIntent().projectTitle().title());
+              Icon book = styleIcon(VaadinIcon.NOTEBOOK);
+
+              projectTitle.add(book, projectCode, projectName);
+            });
+  }
+
+  private Icon styleIcon(VaadinIcon vaadinIcon) {
+    Icon icon = vaadinIcon.create();
+    icon.addClassName("primary");
+    icon.setSize("1em");
```

Comment:
NIT: 
Could be replaced with a css classname to make it reusable in other context (e.g. icon.smallest) 
```suggestion
    icon.setSize("1em");
```